server/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -179,9 +185,23 @@ class WorkflowActorSpec extends CromwellTestKitWordSpec with WorkflowDescriptorB
       finalizationProbe.expectNoMessage(AwaitAlmostNothing)
       deathwatch.expectTerminated(actor)
     }
+
+    "log an error when a PBF initialization throws an exception" in {
+      EventFilter.error(start = "Failed to copy workflow log", pattern = ".*Crashing as requested.*", occurrences = 1).intercept {
+        val _ = createWorkflowActor(WorkflowSucceededState)
+      }
+
+    }
   }
 }
 
+class ExceptionJustForThisTest extends Exception("Crashing as requested")
+
+class CrashingPathBuilderFactory() extends PathBuilderFactory {
+  override def withOptions(options: WorkflowOptions)(implicit as: ActorSystem, ec: ExecutionContext): Future[PathBuilder] =
+    Future(throw new ExceptionJustForThisTest)
+}
```

Comment:
This isn't really crashing so much as politely returning a failed `Future`. But since you've written code to handle misbehaved PBFs that throw rather than return failed `Future`s can you test that too? 🙏

Reply:
Yes please!

Reply:
I don't think the "Paranoia" commit actually does anything because if the PBF throws, we won't reach the handling code.

My evidence for the above is that the stack trace from the throwing PBF in the ticket shows the farthest line reached in `WorkflowActor.scala` is `436`.

Reply:
It appears to be calling back to the `def` which would be protected by paranoia however?
```
at cromwell.core.path.PathBuilderFactory$.instantiatePathBuilders(PathBuilderFactory.scala:23)
	at cromwell.engine.EngineFilesystems$.pathBuildersForWorkflow(EngineFilesystems.scala:27)
	at cromwell.engine.workflow.WorkflowActor$$anonfun$8.bruteForcePathBuilders$1(WorkflowActor.scala:432)
	at cromwell.engine.workflow.WorkflowActor$$anonfun$8.applyOrElse(WorkflowActor.scala:436)
	at cromwell.engine.workflow.WorkflowActor$$anonfun$8.applyOrElse(WorkflowActor.scala:407)
```

I would say it's now an unlikely case since you've hopefully fixed the underlying problem, but defensively having a test for "what happens if the PBF throws an exception" and adding that "what if it throws" paranoia _somewhere_ would still be a good thing IMO